### PR TITLE
Fix errors in sdk-test in presubmit tests

### LINKED ISSUE #351: There are some errors in the sdk-test component of CI/CD tests
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

There are some errors in the sdk-test component of CI/CD tests

```
_____________ ERROR collecting test/test_v1alpha2_xg_boost_spec.py _____________
/usr/local/lib/python3.5/dist-packages/_pytest/python.py:501: in _importtestmodule
   mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/local/lib/python3.5/dist-packages/py/_path/local.py:668: in pyimport
   __import__(modname)
<frozen importlib._bootstrap>:969: in _find_and_load
   ???
<frozen importlib._bootstrap>:958: in _find_and_load_unlocked
   ???
<frozen importlib._bootstrap>:673: in _load_unlocked
   ???
/usr/local/lib/python3.5/dist-packages/_pytest/assertion/rewrite.py:140: in exec_module
   exec(co, module.__dict__)
test_v1alpha2_xg_boost_spec.py:32: in <module>
   import kfserving
../kfserving/__init__.py:17: in <module>
   from .server import KFServer
E     File "/mnt/test-data-volume/kubeflow-kfserving-presubmit-e2e-350-b98d83f-6736-5641/src/kubeflow/kfserving/python/kfserving/kfservin
g/server.py", line 62
E       self.registered_models: Dict[str, KFModel] = {}
E                             ^
E   SyntaxError: invalid syntax
```

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:
Fix errors in sdk-test in presubmit tests

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #351 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/352)

### CODE REVIEW

File: test/scripts/sdk-test.sh [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+
+# Install requirement pacakges.
+python3 -m pip install --upgrade pip
+pip3 install --upgrade pytest
+pip3 install -r python/kfserving/requirements.txt
+
+# Run KFServing SDK unit tests
 pushd python/kfserving/test >/dev/null
-  pytest
+  pytest --ignore=test_server.py --ignore=test_set_creds.py
```
Comment (yuzisun) [inline test/scripts/sdk-test.sh]: Why skipping these two tests?

Comment (jinchihe) [reply test/scripts/sdk-test.sh]: @yuzisun that's not unit test, I think that's need to submit to kubeflow-ci test cluster using gcloud. I will inestigate and add one real test case by this way (gcloud submit) for executing client notebook sample in #353 , shall we enable this then while adding this case? Thanks.

Comment (jinchihe) [reply test/scripts/sdk-test.sh]: @yuzisun As we talked offline, enabled `test_server.py`. Thanks.

**Comment by jinchihe** [comment]
Removed below files since the changes in the PR #350 
```
python/kfserving/test/test_v1alpha2_alibi_explain_spec.py
python/kfserving/test/test_v1alpha2_explain_spec.py
python/kfserving/test/test_v1alpha2_model_spec.py
```

**Comment by jinchihe** [comment]
@yuzisun Enabled `test_server.py` and removed some out-of-date files. That's OK now, please review, thanks.

**Comment by jinchihe** [comment]
@yuzisun Yes, we can test that in design e2e test, for e2e tests, we need to ensure the kfserving is installed in tests env, so that we can create/patch/delete kfservice. The e2e test has nothing now as below.
https://github.com/kubeflow/kfserving/blob/master/test/scripts/run-tests.sh#L23

Shall we enable this while make e2e tests in #353 ? I appended in that ticket to avoid missing. we can consider later for moving or not since this is for kfservice sdk tests.

**Pull request merged**